src/common/engine.js

```diff
@@ -357,7 +364,7 @@ export class CurrencyEngine {
 
       this.transactionListDirty = true
       this.transactionsChangedArray.push(edgeTransaction)
-      this.log.warn('addTransaction new tx: ', edgeTransaction.txid)
+      this.warn('addTransaction new tx: ' + edgeTransaction.txid)
```

Comment:
Change to:
```js
this.warn(`addTransaction new tx: ${edgeTransaction.txid}`) 
// Or
this.warn(`addTransaction new tx: ${edgeTransaction?.txid}`)
// Or even go super safe and do, but I think it's too much
this.warn(`addTransaction new tx: ${JSON.stringify(edgeTransaction?.txid ?? '')}`)
```

The same for everywhere you use concat strings with '+' for the logs.